EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status. It is thus recommended to open a discussion thread on [the Ethereum Magicians forum] to do this, but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the champion to socialize your EIP and build community consensus around it. You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a technically complex Core EIP will have to be much greater than for a simple ERC. While "community interest" will not be a factor in deciding whether an EIP gets merged as a draft, insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP. 
+
+### Core EIPs
+
+For Core EIPs, given that they require client implementations to be considered **Final** (see "EIPs Process" below), you will need to either provide an implementation for clients or convince clients to implement your EIP. 
```

Comment:
I wonder if we shouldn't make a place for this kind of guidance outside EIP 1, so we can restrict EIP 1 to rules and regulations, rather than recommendations.

Reply:
@Arachnid that was my original intent (I added a page to eips.ethereum.org, see: https://github.com/ethereum/EIPs/pull/1932), but then the concern was it becomes "yet another thing to keep updated". 

Not sure which one is the least worse option: more "soft guidance" in EIP-1, or another documentation page.

Reply:
@Arachnid friendly ping :-) Do you have thoughts re: having this in EIP-1 vs. another thing to maintain? Thanks!

Reply:
I think EIP-1 could be split into two main chapters: 1. Rules 2. Guidance/Recommendations

Reply:
@axic interesting. I like that idea. Do you think it'd be valuable to do this work as a refactor of this PR, or do you think it makes sense to merge a minimal version to expand the championing section and then do a "form refactoring" of EIP-1 in two sections?

Reply:
@MadeofTin @axic @Arachnid it seems that this is addressed in https://github.com/ethereum/EIPs/pull/2147 ? If so, does it make sense to merge both EIPs independently or to try and combine them?

Reply:
@timbeiko , I think combining them is a bad idea, removing the "technically sound" requirement means anyone can get an EIP with correct syntax but gibberish tech logic merged. It is very different from having an EIP with technically correct specs but improvable in time.
So, https://github.com/ethereum/EIPs/pull/2147 is a different topic than the current one.